management/server/http/users_handler.go

```diff
@@ -198,7 +198,9 @@ func (h *UsersHandler) GetAllUsers(w http.ResponseWriter, r *http.Request) {
 	users := make([]*api.User, 0)
 	for _, r := range data {
```

Comment:
I wonder if we can only have one check like this:

```go
if r.NonDeletable {
	continue
}
```

And do not check for `!r.NonDeletable` below at all